vendor/sigs.k8s.io/kustomize/api/internal/plugins/builtinhelpers/builtins.go

```diff
@@ -111,4 +109,7 @@ var TransformerFactories = map[BuiltinPluginType]func() resmap.TransformerPlugin
 	ReplacementTransformer:         builtins.NewReplacementTransformerPlugin,
 	ReplicaCountTransformer:        builtins.NewReplicaCountTransformerPlugin,
 	ValueAddTransformer:            builtins.NewValueAddTransformerPlugin,
+	// Do not wired SortOrderTransformer as a builtin plugin.
```

Comment:
is there anything kubectl needs to do to preserve compatible behavior in response to this change?